aten/src/ATen/native/cuda/ReduceMomentKernel.cu

```diff
@@ -32,6 +32,7 @@ static void std_var_kernel_cuda(TensorIterator& iter, bool unbiased, bool take_s
   AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::BFloat16, iter.dtype(), "std_cuda", [&]() {
     std_var_kernel_impl<scalar_t>(iter, unbiased, take_sqrt);
   });
+  iter.copy_reduction_outputs();
```

Comment:
wait, I thought std_var doesn't have out variants?  Also naming this kernel "std_cuda" seems bad :P.

Reply:
Yeah, I learned that it doesn't have out variant when I was writing the test,  I can remove `copy_reduction_outputs` from here.